test/integration/relations.test.js

```diff
@@ -86,82 +83,42 @@ describe.skip('Relations', function () {
         });
     });
 
-    describe('Many-to-Many: Simple Cases', function () {
-        // Use the default file named after the describe block, would be many-to-many-simple-cases
-        beforeEach(utils.db.init());
-
-        afterEach(utils.db.reset());
-
-        it('can match array in (single value)', function (done) {
-            const queryJSON = {'tags.slug': {$in: ['animal']}};
+    describe('Many-to-Many', function () {
+        before(utils.db.init('relations1'));
 
-            // Use the queryJSON to build a query
-            const query = makeQuery(queryJSON);
+        describe('OR', function () {
+            it('tags.slug IN (animal)', function () {
+                const mongoJSON = {'tags.slug': {$in: ['animal']}};
+                const query = makeQuery(mongoJSON);
 
-            // Check any intermediate values
-            console.log(query.toQuery());
-
-            // Perform the query against the DB
-            query.select()
-                .then((result) => {
-                    console.log(result);
-
-                    result.should.be.an.Array().with.lengthOf(3);
-
-                    // Check we get the right data
-                    // result.should.do.something;
-
-                    done();
-                })
-                .catch(done);
-        });
-
-        it('can match array in (multiple values)', function (done) {
-            const queryJSON = {'tags.id': {$in: [2, 3]}};
-
-            // Use the queryJSON to build a query
-            const query = makeQuery(queryJSON);
-
-            // Check any intermediate values
-            console.log('query', query.toQuery());
-
-            // Perform the query against the DB
-            query.select()
-                .then((result) => {
-                    console.log(result);
-
-                    result.should.be.an.Array().with.lengthOf(4);
-
-                    // Check we get the right data
-                    // result.should.do.something;
-
-                    done();
-                })
-                .catch(done);
-        });
-
-        it('can match array in (multiple values with x, y and xy)', function (done) {
-            const queryJSON = {'tags.id': {$in: [1, 2]}};
-
-            // Use the queryJSON to build a query
-            const query = makeQuery(queryJSON);
-
-            // Check any intermediate values
-            console.log('query', query.toQuery());
+                return query
+                    .select()
+                    .then((result) => {
+                        result.should.be.an.Array().with.lengthOf(3);
+                    });
+            });
 
-            // Perform the query against the DB
-            query.select()
-                .then((result) => {
-                    console.log(result);
+            it('featured:true AND tags.slug IN (animal)', function () {
+                const mongoJSON = {$and: [{featured: true}, {'tags.slug': {$in: ['animal']}}]};
```

Comment:
Think this PR is good to go. One thing wanted to touch upon was the way we write these mongoJSON statements. It's quite hard to experiment with them when they are just oneliners like this. Think it makes sense to have them like that for simple statements, but would be good to format them in longer form when there are more that one elements :thinking: 
```suggestion
                const mongoJSON = {
                    $and: [{
                        'tags.slug': {
                            $in: ['animal']
                        }
                    }, {
                        featured: true
                    }]
                };
```

Reply:
Not having any specific formatting in mind, just thinking of the readability of statements under test :)

Reply:
👍